internal/bminventory/inventory.go

```diff
@@ -237,6 +261,96 @@ const nodeIgnitionFormat = `{
   }
 }`
 
+const configStaticIpsScript = `
+#!/bin/bash
+
+function create_template_file() {
+    cat > "/var/tmp/template.connection" <<EOF
+[connection]
+id=\$FOUND_INTERFACE
+interface-name=\$FOUND_INTERFACE
+type=ethernet
+multi-connect=3
+autoconnect=true
+autoconnect-priority=1
+
+[ethernet]
+mac-address-blacklist=
+
+[ipv4]
+method=manual
+addr-gen-mode=eui64
+addresses=\$FOUND_IP/\$FOUND_MASK
+gateway=\$FOUND_GW
+dns=\$FOUND_DNS
+
+[ipv6]
+method=auto
+addr-gen-mode=eui64
+
+[802-3-ethernet]
+mac-address=\$FOUND_MAC
+EOF
+}
+
+function find_my_mac() {
+    MAC_TO_CHECK=${1}
+
+    for entry in $(cat "/var/tmp/static_ips_config.csv")
+    do
+        MAC=$(echo ${entry} | cut -f1 -d\;)
+        if [[ ! -z ${MAC} ]] && [[ -z ${FOUND_MAC} ]]; then
+            if [[ "${MAC}" == "${MAC_TO_CHECK}" ]]; then
+                export FOUND_INTERFACE=${INTERFACE}
+                export FOUND_MAC=${MAC}
+                export FOUND_IP=$(echo ${entry} | cut -f2 -d\;)
+                export FOUND_MASK=$(echo ${entry} | cut -f3 -d\;)
+                export FOUND_DNS=$(echo ${entry} | cut -f4 -d\;)
+                export FOUND_GW=$(echo ${entry} | cut -f5 -d\;)
+                break
+            fi
+        fi
+    done
+
+    if [[ -z ${FOUND_MAC} ]]; then
+        echo "Host MAC ${MAC_TO_CHECK} not found in the list" | systemd-cat -t configure-static-ip -p err
+    fi
```

Comment:
lets print if we did find?

Reply:
done

Reply:
added debug for the next comment, it should cover this case also